r/reward_entry/whitelist.gno

```diff
@@ -0,0 +1,17 @@
+package reward_entry
+
+import "std"
+
+// XXX: Update as required
+var whitelist = []string{
+	"g1jg8mtutu9khhfwc4nxmuhcpftf0pajdhfvsqf5",
```

Comment:
do we already have an admin address shared with the competition operators? if not we should set it now.